pkg/updatestatus/nodeinformer.go

```diff
@@ -40,6 +40,9 @@ type nodeInformerController struct {
 	// sendInsight should be called to send produced insights to the update status controller
 	sendInsight sendInsightFn
 
+	// machineConfigPoolSelectorCache caches the label selectors converted from the node selectors of the machine config pools by their names.
+	machineConfigPoolSelectorCache machineConfigPoolSelectorCache
```

Comment:
consider calling the member just `mcpSelectors` (type name is good though)

Reply:
Done.

https://github.com/openshift/cluster-version-operator/pull/1164/commits/a81ec69e31f504ce10a44650cfa5c4d4260fd2cd#diff-c5a41cb83200e13544e0509a5e77c180e16a1d70d7411a79cac2bc107e89950fR43